src/DataLoaders.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import numpy as np
+import biotite
+import biotite.structure as struc
+import src.DataProcessing  as dp
+from progressbar import ProgressBar
```

Comment:
I recommend using the library [tqdm](https://tqdm.github.io/) for progress bars 😉